tests/integration-tests/README.md

```diff
@@ -786,22 +786,32 @@ def vpc(cfn_stacks_factory):
 Performance tests can run together with functionality tests, reusing clusters created by functionality test.
 
 To run performance test for a specific integration test, the following changes are required:
-1. Add `run_benchmarks` fixture to the test:
+1. Add `run_benchmarks` and `benchmarks` fixtures to the test:
 ```python
-def test_ad_integration(..., run_benchmarks):
+def test_ad_integration(..., run_benchmarks, benchmarks):
```

Comment:
I think we might want to call this `test_feature` rather than `test_ad_integration` so that the README is generic.

Reply:
Done